toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -57,13 +58,29 @@ func ConvertNodesToRequests(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMute
 		requests = append(requests, req)
 	}
 
-	requests = append(requests, buildNodesToRequests(pkgGraph, buildState, packagesToRebuild, testsToRerun, buildNodes, isCacheAllowed)...)
-	requests = append(requests, testNodesToRequests(pkgGraph, buildState, testsToRerun, testNodes)...)
+	newBuildReqs, err := buildNodesToRequests(pkgGraph, buildState, packagesToRebuild, testsToRerun, buildNodes, isCacheAllowed)
+	if err != nil {
+		err = fmt.Errorf("failed to convert build nodes to requests:\n%w", err)
+		// Temporarily clear the error, this state is unexpected but not fatal. Error return will be
+		// restored later once the underlying cause of this error is fixed.
+		err = nil
+		//return
+	}
+	requests = append(requests, newBuildReqs...)
+	newTestReqs, err := testNodesToRequests(pkgGraph, buildState, testsToRerun, testNodes)
+	if err != nil {
+		err = fmt.Errorf("failed to convert test nodes to requests:\n%w", err)
+		// Temporarily clear the error, this state is unexpected but not fatal. Error return will be
+		// restored later once the underlying cause of this error is fixed.
+		err = nil
+		//return
+	}
```

Comment:
Witha second look, if we are squelching the error in the inside loop we don't need to mess with these bits.
```suggestion
	if err != nil {
		err = fmt.Errorf("failed to convert test nodes to requests:\n%w", err)
		return
	}
```